client/src/components/Notifications/NotificationDropDown.js

```diff
@@ -231,8 +265,10 @@ const menu = (notifications, organisationId, t) => {
       </Menu.Item>
       <div className="notifications-container">
         <div>
-          {notifications.map((each) => (
+          {notifications.map((each, i) => (
```

Comment:
Rather than use a one letter variable name, which is hard to read, I would change this to `index`